pkg/gofr/container/container.go

```diff
@@ -121,7 +119,7 @@ func (c *Container) Create(conf config.Config) {
 		c.PubSub = c.createMqttPubSub(conf)
 	}
 
-	c.File = file.New(c.Logger)
+	//c.File = file.New(c.Logger)
```

Comment:
This change look awkward, like a left behind debug or a code that was forgotten to be cleaned